dicom_app/dicom_reader.py

```diff
@@ -0,0 +1,25 @@
+import pydicom
+from pydicom.data import get_testdata_files
+
+from PIL import Image
+from PyQt5.QtGui import QImage
+import matplotlib.pyplot as plt
+
+
+def convert_dicom_to_QImage(cvImg):
+    height, width = cvImg.shape
+    bytesPerLine = 3 * width
```

Comment:
bytesPerLine => bytes_per_line
Don't use CamelCase, use snake_case. PyQT uses camelcase ... (weird)

Reply:
More specifically it is mixedCase